content/sensu-go/5.18/getting-started/enterprise.md

```diff
@@ -40,8 +48,11 @@ Manage your Sensu account and contact support through [account.sensu.io][2].
 
 If you haven't already, [install the Sensu Go backend, agent, and sensuctl tool][15] and [configure sensuctl][16].
 
-Log in to your Sensu account at [account.sensu.io][2] and download your commercial license file.
-Click **Download license**.
+You will need a commercial license if your Sensu installation includes more than 100 entities.
+To download your commercial license file:
+
+1. Log in to your Sensu account at [account.sensu.io][2].
+2. Click **Download license**.
```

Comment:
Note: In some circumstances you may need to click "generate license" before you can download a license.

![image](https://user-images.githubusercontent.com/148017/77211578-05b66900-6aca-11ea-93cc-4bfb32dfddab.png)